modules/issue_tracker/jsx/columnFormatter.js

```diff
@@ -95,6 +133,9 @@ function formatColumn(column, cell, rowData, rowHeaders) {
       </td>
     );
   }
+  if (column === 'Watching') {
+    return (<WatchingCheckbox checked={cell==="1"} issue_id={row['Issue ID']}/>);
```

Comment:
You should wrap the checkbox inside `<td>` here instead of directly inside the component. 
This is more explicit and it will make it easier to get rid of these `<td>`s in the future